docs/build/isc/v1.0.0-rc.6/sidebars.js

```diff
@@ -159,8 +159,12 @@ module.exports = {
             {
               type: 'category',
               label: 'Token',
-              link: { type: 'doc', id: 'how-tos/token/introduction' },
               items: [
+                {
```

Comment:
You don‘t like linked categories? 😢 😆

Reply:
I do, but I'm certain we had an issue with them with this version of docusarus and I had to remove them

Reply:
We can leave them if you like, if the issue comes back we can remove it :)

Reply:
It was only related to use the introduction page both as the first page AND a category link. So both doesn't work.
Se this issue: https://github.com/iotaledger/hornet/issues/1825#issuecomment-1538487840

Reply:
So should we not use category pages? In the end I always saw category pages as something that you don't need while going through the sidebar but is nice to link to.

Reply:
I think we can use them, but not as a link.

Reply:
That is a no. Because if I talk about category page I mean using `link: ...`